### ISSUE TITLE
Audio delay

### ISSUE THREAD

**Issue by Rex000** [xenia-canary/xenia-canary#201]
### Validation

- [X] I've read the [FAQ](https://github.com/xenia-project/xenia/wiki/FAQ).
- [X] The Xenia build used is from the master branch (not MLBS/AlexVS/Canary/pull requests, etc.)
- [X] This issue isn't for tech support (help with Xenia).
- [X] If this issue occurs in a specific game, I've done analysis to locate the faulty subsystem of the emulator and a potential reason in it.
- [X] I've checked if this issue hasn't already been reported.
- [X] My device meets the minimum requirements: https://github.com/xenia-project/xenia/wiki/Quickstart#system-requirements
- [X] (If building) I have read the building doc: https://github.com/xenia-project/xenia/blob/master/docs/building.md

### Describe what's going wrong

There is a significant amount of audio delay in every game that I have tried in the latest build and even in the August build. I don't know since when this issue started occurring with canary but I came to know that it's an issue with master as well. Even setting the "apu_max_queued_frames" value to "16" which is stated as the lowest possible barely improves it.

### Describe what should happen

No audio delay

### If applicable, provide a callstack here, especially for crashes

N/A

### If applicable, upload a logfile and link it here

N/A

**Comment by laurac8r** [comment]
Try
```shell
apu = "sdl"
apu_max_queued_frames = 16
```
in xenia-canary.config.toml

Credit:

1. https://www.reddit.com/r/emulation/comments/sjpflh/comment/hvn32zb/?utm_source=share&utm_medium=web2x&context=3
2. https://github.com/xenia-canary/xenia-canary/blob/d84df6e47f5338f0d0cff36bb08c4d27d6c442b4/src/xenia/apu/audio_system.cc#L50

**Comment by ManuDoni** [comment]
> ```shell
> apu_max_queued_frames 
> ```

Seems like apu_max_queued_frames doesn't exist on the latest version (non-canary)

**Comment by svetter** [comment]
Setting `apu_max_queued_frames = 16` definitely helps but there still remains a significant amount of delay. `sdl` doesn't seem to make a difference on my system at least.

**Comment by ParanormalBanana** [comment]
> Setting `apu_max_queued_frames = 16` definitely helps but there still remains a significant amount of delay. `sdl` doesn't seem to make a difference on my system at least.

Exactly. The delay is somewhere between half a second and a full second. How can we fix this?
